Dockerfile

```diff
@@ -1,6 +1,6 @@
-FROM mhart/alpine-node:7
+FROM alpine
 
+RUN apk add --update nodejs nodejs-npm
```

Comment:
I'm fine with this, but can't merge because this PR also contains the example HTML for which I have remarks.
Please try to split your PRs so they can be atomicly reviewed and merged, if possible.